client/my-sites/patterns/hooks/use-patterns.ts

```diff
@@ -17,7 +17,6 @@ export function getPatternsQueryOptions(
 		},
 		staleTime: Infinity,
 		...queryOptions,
-		enabled: !! category,
```

Comment:
How come you removed this? The idea was not to fetch patterns on the `/patterns` home page because we don't render any pattern previews there.

Reply:
Right 😄